addons/web/static/src/js/libs/pdfjs.js

```diff
@@ -18,3 +18,38 @@
 * Finally the option was choosed to modify the default value of this option directly in pdf.js as no hook worked in the
 * 'account_invoice_extract' module.
 */
+odoo.define('web.pdfjs.mixin', function (require) {
+    "use strict";
+
+    const config = require('web.config');
+
+    const hidePDFJSButtons = (rootElement) => {
```

Comment:
personal style: i prefer normal functions in top level file. So, `function hidePDFJSButtons (rootElement) {...)`